From 505de9ecf49cb61a12969ee2743a4544f7e4008e Mon Sep 17 00:00:00 2001
From: Mahesh Bodapati <mbodapat@xilinx.com>
Date: Mon, 23 Jan 2017 15:39:45 +0530
Subject: [PATCH 02/10] Add missing declarations for xil_printf to stdio.h for
 inclusion in toolchain and use in c++ apps

Signed-off-by: David Holsgrove <david.holsgrove@xilinx.com>

Conflicts:
	newlib/libc/include/stdio.h

(cherry picked from commit 765f715f4077780395d381bf25870b61008f8013)
---
 newlib/libc/include/stdio.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/newlib/libc/include/stdio.h b/newlib/libc/include/stdio.h
index 77966578d802..fda52fdba1ec 100644
--- a/newlib/libc/include/stdio.h
+++ b/newlib/libc/include/stdio.h
@@ -251,6 +251,9 @@ int	sprintf (char *__restrict, const char *__restrict, ...)
                _ATTRIBUTE ((__format__ (__printf__, 2, 3)));
 int	remove (const char *);
 int	rename (const char *, const char *);
+void	xil_printf (const char*, ...);
+void	putnum (unsigned int );
+void	print (const char* );
 #ifdef _LIBC
 int	_rename (const char *, const char *);
 #endif
-- 
2.47.3

